cps/web.py

```diff
@@ -1283,7 +1291,7 @@ def get_download_link(book_id, format):
             response.headers["Content-Type"] = mimetypes.types_map['.' + format]
         except:
```

Comment:
Ok, I thought this PR wil hangoff, so i didn't fully follow coding standard.
I will amend all.